src/shappie/interaction.py

```diff
@@ -139,21 +140,33 @@ async def respond_to_message(self) -> dict[str, str]:
             return await self._respond_to_message_without_tools()
 
     async def _chatbot_mode(self):
-        if self._did_mention_bot():
-            async with self._message.channel.typing():
-                results = await self.respond_to_message()
+        mention = self._did_mention_bot()
+        should_respond = self.should_respond()
+        if not mention and not should_respond:
+            return
+        if self._message.channel.id != 1125813800151547974 and self._message.guild.id == 1099335575745613835:
```

Comment:
We should not be hardcoding channel and guild IDs. This needs to be configurable. One option is environment variables for now, but we probably want to start thinking about an actual configuration file. I might also consider making these instance variables on the interaction class, and potentially allowing more than one channel for responses. What if we want to let the bot talk in a few different channels? Perhaps we don't need to implement that now, but at least setting up some instance variables gets us closer to that.

Reply:
I went with a config, but this would be temporary as you'd want to switch the configuration to be inside the db to be accessible from the dashboard, I can do that if you'd like.